iXBRLViewerPlugin/viewer/src/js/ixbrlviewer.js

```diff
@@ -165,52 +165,63 @@ iXBRLViewer.prototype.load = function () {
     setTimeout(function () {
 
         iv._loadInspectorHTML();
-        var iframes = $(iv._reparentDocument());
+        var iframes;
+        const stubViewer = $('body').hasClass('ixv-stub-viewer');
+        if (!stubViewer) {
+            iframes = $(iv._reparentDocument());
+        } 
+        else {
+            iframes = $();
+        }
 
         var taxonomyData = iv._getTaxonomyData();
         if (taxonomyData === null) {
             $('#ixv .loader .text').text("Error: Could not find viewer data");
             $('#ixv .loader').removeClass("loading");
             return;
         }
-        var report = new iXBRLReport(JSON.parse(taxonomyData));
-        if (report.isDocumentSet()) {
-            var ds = report.documentSetFiles();
-            for (var i = 1; i < ds.length; i++) {
-                var iframe = $("<iframe />").attr("src", ds[i]).appendTo("#ixv #iframe-container");
-                iframes = iframes.add(iframe);
-            }
+        const report = new iXBRLReport(JSON.parse(taxonomyData));
+        const ds = report.documentSetFiles();
+        var hasExternalIframe = false;
+        for (var i = stubViewer ? 0 : 1; i < ds.length; i++) {
+            const iframe = $("<iframe />").attr("src", ds[i]).appendTo("#ixv #iframe-container");
```

Comment:
Pre-existing, I think the `src` needs to somehow URL encode path segments.  Otherwise `Q3: Workiva 10-Q.htm` will be mishandled (`Q3` interpreted as a scheme and normalized to `q3`), or a filename like `javascript:alert(1).html` would become active content.

Reply:
(Approved if you'd rather this be fixed separately.)

Reply:
Done in [f440403](https://github.com/Workiva/ixbrl-viewer/commit/f440403b388e326c78eea5dddfec3559478bf4a9)

I think it makes sense to do this in viewer generation, rather than the client, and treat this as a list of valid URLs, as this gives us a bit more flexibility in the future if we wanted to be able to put absolute URLs here.